fog/README.md

```diff
@@ -58,7 +58,10 @@ To run the unittest tests locally, you need to start a local postgres instance,
 ./mob prompt
 
 # Start postgres
-service postgresql start
+sudo service postgresql start
+
+# create a postgres user
+sudo -u postgres createuser --superuser $USER
```

Comment:
why is this not needed for circle ci i wonder?

Reply:
My understanding is because `mob prompt` isn't used in CI.
The recent changes to `mob prompt` set some env variables, https://github.com/mobilecoinfoundation/mobilecoin/blob/master/mob#L211 
These get picked up by https://github.com/mobilecoinofficial/docker-rust-sgx-base/blob/main/entrypoint-builder-install.sh#L20

This means only if those env values are set will the user info propagate into the container, otherwise the containers run as before, which is as sudo

Reply:
can we add this line to the entrypoint.sh instead?

Reply:
@jgreat do you have thoughts on expanding the entrypoint.sh to create a postgres user, since it's looking at the user env already.

Reply:
We could put those pg bootstrapping steps in the entrypoint, but longer term we should look at extracting pg from the mob prompt container and utilize the GHA `services:` options for pg usage.  The pg container does all the boot strapping for you. 

We could extend mob prompt to wrap docker compose for external services required for local development.

Reply:
Actually we already do the user creation in the image build: https://github.com/mobilecoinofficial/docker-rust-sgx-base/blob/06741c8b14c3cd734ecc371874d9aafc272192b9/Dockerfile#L93-L96

Could add `service postgresql start` to the entrypoint.sh

Reply:
that creates the `root` user, we need to create one for the localuser

Reply:
With #2607 and the comment of 
> onger term we should look at extracting pg from the mob prompt container and utilize the GHA services: options for pg usage. The pg container does all the boot strapping for you.

> We could extend mob prompt to wrap docker compose for external services required for local development.

I'm going to resolve and merge this.  Once that longer term solution happens it will require removing both postgres steps from the readme